def exception_function():
    raise RuntimeError('This is a dummy exception')


if __name__ == '__main__':
    exception_function()
